client/src/components/SignInSteps/Step.jsx

```diff
@@ -0,0 +1,80 @@
+import { Form, Formik, ErrorMessage } from "formik";
+import React from "react";
+import Paginations from "./Paginations";
+import "./Step.css";
+import Three from "./StepThree/Three";
+import * as Yup from "yup";
+
+const Step = ({ title, currentStep, handleSteps }) => {
+  const initialValues = {
+    jobChecked: [],
+  };
+  const schemaFormValidation = Yup.object().shape({
+    jobChecked: Yup.array()
+
+      .min(1, "You need to select one job category")
+
+      .max(1, "You can only select one jon category")
+
+      .required("Required"),
+  });
+  function renderForm(currentStep) {
+    switch (currentStep) {
+      case 1:
+        return "Step 1";
+      case 2:
+        return "Step 1";
+      case 3:
+        return <Three />;
+      case 4:
+        return "Step 1";
+      default:
+        return "This step does not exist.";
+    }
+  }
+
+  function onSubmit(values) {
+    const data = JSON.stringify(values, null, 2);
+    console.log(values);
+    // data = {
+    //   "firstName": "sdcqds",
+    //   "lastName": "dcsdqc",
+    //   "email": "csdcsd@dql.com",
+    //   "password": "dqscsdx",
+    //   "newsLetter": true
+    // }
+
+    // axios.post(" https://alunmi-agora-backend.herokuapp.com/api/signup",data).then(id=>);
+    handleSteps();
```

Comment:
how does the values are saved if you don't passes them to the function ?

Reply:
save should work in local and on the server, missing URL configuration here

Reply:
Formik does ? alone ? but when I console log the values I do have access to the Job Category selected

Reply:
saved in DB I mean

Reply:
Oh yeah, did not implement that because the PR was already big and late. Wanted to share step 3 and 4  the do new branches/PR to link with database